### ISSUE TITLE
Panicked again on Linux

### ISSUE THREAD

**Issue by Tenyun** [tramhao/termusic#114]
Hi,

after fix #106 I run into a panic again.
I have tested both the master, the dev version (make full) and the release download and always get the same error message.
I am on Archlinux.

```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/config/mod.rs:65:24
stack backtrace:
   0:     0x55cdb5a0ff4a - <unknown>
   1:     0x55cdb5a391ee - <unknown>
   2:     0x55cdb5a08fe5 - <unknown>
   3:     0x55cdb5a0fd15 - <unknown>
   4:     0x55cdb5a1169f - <unknown>
   5:     0x55cdb5a113db - <unknown>
   6:     0x55cdb5a11dac - <unknown>
   7:     0x55cdb5a11b02 - <unknown>
   8:     0x55cdb5a103fc - <unknown>
   9:     0x55cdb5a11852 - <unknown>
  10:     0x55cdb4f6fe43 - <unknown>
  11:     0x55cdb4f6ff1d - <unknown>
  12:     0x55cdb4f2cc23 - <unknown>
  13:     0x55cdb5194a19 - <unknown>
  14:     0x55cdb519f8c4 - <unknown>
  15:     0x55cdb4fa5783 - <unknown>
  16:     0x55cdb50bcbad - <unknown>
  17:     0x55cdb5a02ffc - <unknown>
  18:     0x55cdb51a54f5 - <unknown>
  19:     0x7fd92d506790 - <unknown>
  20:     0x7fd92d50684a - __libc_start_main
  21:     0x55cdb4f7046e - <unknown>
  22:                0x0 - <unknown>
```

**Comment by FlanOfHarmony** [comment]
Same on Fedora 37

**Comment by tramhao** [comment]
My bad. There are two places but I only changed one. Will fix right now.

**Comment by tramhao** [comment]
Please check out master branch: b6006b22901f1f865a2e3acf7490fd3fa520ca5e .
Sorry for the mistake.

**Comment by Tenyun** [comment]
Application works.
I only had a few minutes to test it this morning.
I have already found two new problems but I need to make sure they are not on my side.
I also have some suggestions for improvements that I will put in a new issue.

Thanks, keep up the good work.

**Comment by saponciou** [comment]
Im still getting the error on arch linux

**Comment by tramhao** [comment]
This should be solved in 0.7.10.
